sensors/06-list-datasets-by-plot.Rmd

```diff
@@ -66,76 +58,109 @@ files = extract_file_paths(datasets)
 Datasets can be further filtered using the *since* and *until* parameters
 of `get_file_listing` with a date string.
 
-```{python}
+```{python eval=FALSE}
 dataset = get_file_listing(None, url, key, sensor, sitename, 
         since='2016-06-01', until='2016-06-10')
 ```
 
 
-# Alternative method
+# Querying the API
+
+The source files behind the data are available for downloading through the API. By executing a series
+of requests against the API it's possible to determine the files of interest and then download them.
+
+Each of the API URL's have the same beginning (https://terraref.ncsa.illinois.edu/clowder/api), 
+followed by the data needed for a specific request. As we step through the process you will be able
+to see how then end of the URL changes depending upon the reuqest.
+
+Below is what the API looks like as a URL. Try pasting it into your browser.
+
+https://terraref.ncsa.illinois.edu/clowder/api/geostreams/sensors?sensor_name=MAC Field Scanner Season 1 Field Plot 101 W
+
+This will return data for the requested plot including its id. This id (or identifier) can then be used for 
+additional queries against the API.
+
+In the examples below we will be using **curl** on the command line to make our API calls. Since the
+API is accessed through URLs, it's possible to use the URLs in software programs or with a programming language
+to retrieve its data. 
 
-The following method demonstrates the same approach using the Clowder API. This
-approach is useful for understanding the data layout and when the Python
-terrautils package is not available.
+## A Word of Caution
+
+The names of variables in this section don't necessarily match the ones in the section 
```

Comment:
Hmmm ... not sure how to best put this. Explain that in this section you are querying the API directly. Unlike the terrautils client, the API endpoints have names based on the database structure.

Reply:
Made changes